Cargo.lock

```diff
@@ -3264,6 +3380,15 @@ dependencies = [
  "webpki",
 ]
 
+[[package]]
+name = "webpki-roots"
+version = "0.20.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "0f20dea7535251981a9670857150d571846545088359b28e4951d350bdaf179f"
+dependencies = [
+ "webpki",
+]
```

Comment:
Ditto, it's surprising this is being updated

Reply:
Thanks @bartlomieju 🙃